OpenRA.Mods.Common/Activities/Move/AttackMoveActivity.cs

```diff
@@ -18,31 +19,80 @@ namespace OpenRA.Mods.Common.Activities
 {
 	public class AttackMoveActivity : Activity
 	{
-		const int ScanInterval = 7;
-
+		readonly Func<Activity> getInner;
 		Activity inner;
-		int scanTicks;
+		Activity attack;
 		AutoTarget autoTarget;
+		bool moving;
+		readonly bool assaultMoving;
```

Comment:
if you make this `public readonly bool IsAssaultMoving;` we can avoid the overhead of resolving the property indirection.

Reply:
Also beware the stylecop failure here.

Reply:
Fixed on both accounts.

Also, is there a way to run make check only for a single assembly?

Reply:
`make check` just runs `OpenRA.StyleCheck.exe` on each project directory in turn - you can run it directly on e.g. the `OpenRA.Mods.Common` dir if you want to check just a single assembly.

Reply:
Thanks!